deutsches-archaologisches-institut.csl

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<style class="in-text" version="1.0" delimiter-precedes-et-al="never" initialize-with="." demote-non-dropping-particle="sort-only" default-locale="de" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/#) -->
   <info>
     <title>Deutsches Archäologisches Institut</title>
```

Comment:
Since you're switching the style to German, could you change the title to `Deutsches Archäologisches Institut (German)` ? No need to change id/filename etc.